### DISCUSSION TITLE
not able to power on certain stb

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#202] [General]
I helped a friend set up slinger on his pc.  It worked except that the power button on the remote will not power on the stb.  It will power off the stb, though.  He can also power it on with the virtual remote on a web slingplayer.  What can we do in this case?

Thanks.

**Comment by GerryDazoo** [comment]
Yes, some boxes have separate keycodes for power on and power off. You'll have to figure out what keycode to send for power-on.
See if you can find his remote here.
http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=98
